ibstore/_store.py

```diff
@@ -454,25 +522,47 @@ def optimize_mm(
             chunksize=chunksize,
         )
 
-        for result in _minimized_blob:
-            inchi_key = result.inchi_key
-            molecule_id = self.get_molecule_id_by_inchi_key(inchi_key)
-            self.store_conformer(
-                MMConformerRecord(
+        inchi_to_id = {
```

Comment:
(if correct) this would help me mentally keep track of things
```suggestion
        inchi_to_id: dict[int, str] = {
```